suffuse/node_fuse.go

```diff
@@ -3,22 +3,92 @@ package suffuse
 import (
   "github.com/suffuse/go-suffuse/suffuse/xattr"
   "os"
+  "time"
   "golang.org/x/net/context"
   "bazil.org/fuse/fs"
   f "bazil.org/fuse"
 )
 
+// Access checks whether the calling context has permission for
```

Comment:
In other places you do use `/*`, I find those much more readable, less noise.

We could go for a separation:

- Comments that explain something and will live as long as the method itself go in multiline
- Comments that are likely to be removed in the short run (for example missing feature notes) go in single line